build

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+
+# build - build Jellyfin binaries or packages
+
+set -o errexit
+set -o pipefail
+
+# The list of possible package actions (except 'clean')
+actions=( 'build' 'package' 'sign' 'publish' )
+
+# The list of possible platforms, based on directories under 'deployment/'
+platforms=( $(
+    find deployment/ -maxdepth 1 -mindepth 1 -type d | sed 's/deployment\///' | sort
+) )
+
+# The list of standard dependencies required by all build scripts; individual
+# action scripts may specify their own dependencies
+dependencies=( 'tar' 'zip' )
+
+usage() {
+    echo -e "build - build Jellyfin binaries or packages"
+    echo -e ""
+    echo -e "Usage:"
+    echo -e " $ build --list-platforms"
+    echo -e " $ build --list-actions <platform>"
+    echo -e " $ build [-b/--web-branch <web_branch>] <platform> <action>"
+    echo -e ""
+    echo -e "The web_branch defaults to 'master'."
+    echo -e "To build all platforms, use 'all'."
+    echo -e "To perform all build actions, use 'all'."
+    echo -e "Build output files are collected at '../jellyfin-build/<platform>'."
+}
+
+if [[ -z $1 ]]; then
+    usage
+    exit 1
+fi
+
+# Verify required utilities are installed
+for utility in ${dependencies[@]}; do
+    if ! which ${utility} &>/dev/null; then
+        echo -e "This script requires the '${utility}' utility."
```

Comment:
shouldn't the script exit if required dependency is not there?

Reply:
I think this is the only blocker for me, others are `nice-to-have`-s.

Reply:
Yes yes it should ;-)